### PR TITLE
Add inhomogeneous Poisson generator NESTML model

### PR DESCRIPTION
**Pull request by clinssen**

### CODE REVIEW

**Review by pnbabu — CHANGES_REQUESTED** [review]
This is a really nice tutorial, thanks for adding it! I have some minor comments.

File: pynestml/symbols/predefined_functions.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ class PredefinedFunctions:
         LOGGER_INFO           The callee name of the logger-info function.
         LOGGER_WARNING        The callee name of the logger-warning function.
         RANDOM_NORMAL         The callee name of the function used to generate a random normal (Gaussian) distributed variable with parameters `mean` and `var` (variance).
+        RANDOM_POISSON        The callee name of the function used to generate a random Poissonian distributed variable with parameter `lmbda` (expected value).
```
Comment (pnbabu) [inline pynestml/symbols/predefined_functions.py:48]: Could you also add this to the language docs?

**Comment by clinssen** [comment]
Thanks for the review! I have implemented all of your comments. Could you have another look?

**Review by pnbabu — APPROVED** [review]
Looks good to me!

**Comment by clinssen** [comment]
Cheers!

**Pull request merged**